packages/services/f-braze-adapter/src/services/_tests/BrazeConsumer.test.js

```diff
@@ -0,0 +1,133 @@
+import BrazeConsumer from '../BrazeConsumer';
+import { removeDuplicateContentCards } from '../utils';
+import transformCardData from '../utils/transformCardData';
+import { LogService } from '../logging/logging.service';
+
+const mockCards = [
+    {
+        title: '51 Pegasi b',
+        extras: {
+            updated: '2020-02-17T13:23:58.000Z',
+            custom_card_type: 'Promotion_Card_1', // eslint-disable-line camelcase
+            order: '2'
+        }
+    },
+    {
+        title: 'Wasp-17b',
+        extras: {
+            updated: '2020-02-16T12:28:58.000Z',
+            custom_card_type: 'Promotion_Card_2', // eslint-disable-line camelcase
+            order: '3'
+        }
+    },
+    {
+        title: 'Wasp-19b',
+        extras: {
+            updated: '2020-02-15T18:23:58.000Z',
+            custom_card_type: 'Promotion_Card_3', // eslint-disable-line camelcase
+            order: '1'
+        }
+    }
+];
+
+const mockConsumerOptions = {
+    enabledCardTypes: [
+        'Home_Promotion_Card_1',
+        'Home_Promotion_Card_2',
+        'Post_Order_Card_1',
+        'Promotion_Card_1',
+        'Promotion_Card_2'
+    ],
+    callbacks: {
+        contentCards: jest.fn()
+    },
+    interceptInAppMessages: {
+        messages: jest.fn()
+    },
+    interceptInAppMessageClickEvents: {
+        clickEvents: jest.fn()
+    },
+    loggerCallbacks: {
+        logger: jest.fn()
+    },
+    brands: [
+        'a',
+        'b',
+        'c',
+        'd'
+    ]
+};
+
+describe('BrazeConsumer', () => {
+    it.each`
```

Comment:
Is this a typo or a template literal?

Reply:
It's a template literal, it's already a pattern in use though so we'll leave it and have a discussion on their use later.